proto/spire/api/server/svid/v1/svid.proto

```diff
@@ -125,6 +126,9 @@ message NewDownstreamX509CAResponse {
 
     // X.509 authorities (DER encoded).
     repeated bytes x509_authorities = 2;
+
+    // X.509 authorities with metadata
+    repeated spire.api.types.X509Certificate x509_authorities_with_metadata = 3;
```

Comment:
this is too be sure that on initial downstream call, we get the list of tainted keys

Reply:
all of them has different uses, 
`NewDownstreamX509CAResponse`: upstream authority is the one communicating with the Upstream Authority, so that plugin can call getBundle from Upstream and get the list of affected bundles, and propagate that information to downstream.
so we can taint the Downstream bundle that was signed by an unsecure authority.

So this is to get the bundle when communication between plugin and agent started after that we can rely on getbundle to verify if there are updates

Reply:
in the end the list of bundles and state is returned to ca manager (on server)  for processing and rotate if required

Reply:
I still wonder if there's some way around having to do this .. I spent a little time looking at the upstream authority plugin code, and it appears that we get root keys from JWT response, X509 response, and then combine them to form the bundle that should be propagated downstream. And then that causes us to have to poke these taint bits through

One option could be to change the plugin logic to load from the upstream bundle API sooner, and use it purely as the source of truth (rather than use JWT + X509 roots as source of truth on boot, then change over to bundle API). That way, the only place we communicate taint data is through the bundle.

I don't think this is blocking though, just an observation.